### PR TITLE
Spread config in corresponding modules

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Gone with the config file and welcome to Jon's `getBaseApiUrl` (ex-`deriveEndpoint`).

### CODE REVIEW

File: src/_shared/js/dev.js [hunk 1]
```diff
@@ -1,4 +1,12 @@
 // necessary when running multiple local services and making CORS requests
 export function portify(host, target = '9000') {
-  return host.replace('7000', target);
+    return host.replace('7000', target);
+}
+
+let apiBaseUrl = 'https://api.nextgen.guardianapps.co.uk';
+
+/*  The PROD endpoint for Soulmates is not on theguardian.com, so we must detect
+    whether we are in DEV or PROD before supplying the full endpoint */
```
Comment (JonNorman) [inline src/_shared/js/dev.js]: This comment could do with being updated / made generic.

Comment (JonNorman) [inline src/_shared/js/dev.js]: And shall we move this into `utils.js`?

**Comment by Ap0c** [comment]
Looks good! Only one minor point, there's still a `config.js` dependency in the blended template causing a build warning.

**Comment by regiskuckaertz** [comment]
Yikes! Thanks @Ap0c :skull_and_crossbones:

File: src/_shared/js/dev.js (hunk 1 above)
Comment (regiskuckaertz) [inline src/_shared/js/dev.js]: Not yet 🙈

**Pull request merged**